`AssertPFSHistory` failes in PFS7 scenario

The nodes seem to behave correctly, but the asert expects that there are exactly two routes, while there are two routes per response. Here's the output from the debug endpoint:

```json
{
  "request_count": 10,
  "responses": [
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    },
    {
      "source": "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
      "target": "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b",
      "routes": [
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0xfED2e042aBB26EbaBD0a11843Caf26922994a8dE",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        },
        {
          "path": [
            "0xc6b8CBfa3b233cc5787Ff14D1457e8F42545B68e",
            "0x41b6f82A50AdDB2ff6A17d85Ce32f7a83f6e1B5c",
            "0x6A1CE7838824989331cf29C143d9E74fAB53C990",
            "0x4113E650E5474A2EAa8942D9d5f72B5664C9D97b"
          ],
          "estimated_fee": 0
        }
      ]
    }
  ]
}
```

It's the last assertion in the scenario:
```
Traceback (most recent call last):
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/base.py", line 74, in __call__
    return self._run(*args, **kwargs)
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/execution.py", line 41, in _run
    task()
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/base.py", line 74, in __call__
    return self._run(*args, **kwargs)
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/execution.py", line 41, in _run
    task()
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/base.py", line 74, in __call__
    return self._run(*args, **kwargs)
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/api_base.py", line 70, in _run
    return self._process_response(response_dict)
  File "/home/karl/scenario-player/venv/lib/python3.7/site-packages/scenario_player/tasks/services.py", line 205, in _process_response
    f"Expected {len(exp_routes)} routes but got {len(actual_routes)}."
scenario_player.exceptions.legacy.ScenarioAssertionError: Expected 2 routes but got 166.
```

--- a/raiden/tests/scenarios/pfs7_multiple_payments.yaml
+++ b/raiden/tests/scenarios/pfs7_multiple_payments.yaml
@@ -10,8 +10,8 @@ settings:
       enable: true
       token:
         deposit: true
-        balance_per_node: 20000
-        min_balance: 20000
+        balance_per_node: 7_000_000_000_000_000_000
+        min_balance: 5_000_000_000_000_000_000
 
 token:
 
@@ -66,7 +66,7 @@ scenario:
             - assert: {from: 3, to: 4, total_deposit: 1000, balance: 1000, state: "opened"}
       - serial:
           name: "Make 100 transfers from 0 to 3"
-          repeat: 100 
+          repeat: 100
           tasks:
             # An amount is used that eventually cause one path to run out of capacity
             # and must then use the other path instead.
@@ -84,8 +84,9 @@ scenario:
             - assert_pfs_iou: {source: 0, amount: 10000}
             - assert_pfs_history:
                 source: 0
-                request_count: 100 
                 target: 3
+                request_count: 100
+                distinct_routes_only: true
                 expected_routes:
                   - [0, 4, 3]
                   - [0, 1, 2, 3]

